proto/sentry_protos/snuba/v1/endpoint_trace_item_stats.proto

```diff
@@ -10,6 +10,8 @@ message AttributeDistributionsRequest {
   // max_buckets and limit wouldn't apply to other stats types (like totals)
   uint32 max_buckets = 1;
   uint32 max_attributes = 2;
+  // if specified, only the attributes in the allow list will be returned
+  repeated AttributeKey attribute_allow_list = 3;
```

Comment:
We call this simply `attributes` in the `GetTrace` endpoint, I'd keep it the same. Also, do we need to limit this to a specific item type as well?

https://github.com/getsentry/sentry-protos/blob/main/proto/sentry_protos/snuba/v1/endpoint_get_trace.proto#L18